ocpp1.6_test/proto_test.go

```diff
@@ -157,5 +157,5 @@ func (suite *OcppV16TestSuite) TestErrorCodes() {
 	t := suite.T()
 	suite.mockWsServer.On("Start", mock.AnythingOfType("int"), mock.AnythingOfType("string")).Return(nil)
 	suite.centralSystem.Start(8887, "somePath")
-	assert.Equal(t, ocppj.FormatViolationV16, ocppj.FormationViolation)
+	assert.Equal(t, ocppj.FormatViolationV16, ocppj.FormatErrorType(suite.centralSystem))
```

Comment:
Wouldn't it be fine to just run the function against `suite.ocppjClient` (or ocppjServer)? These also implement the `dialector` interface, so it would save one more function in the v16/v2 package. All tests should have access to the ocppj structs directly.

Reply:
Brilliant- exactly what I wanted!